src/Components/events/DetailEvent.js

```diff
@@ -20,26 +16,20 @@ const DetailEvent = () => {
   const eventId = useParams().id;
   const location = useLocation();
   const userId = location.state.userId;
+  const eventsList = location.state.eventsList;
   const [goToCreateNewEvent, setGoToCreateNewEvent] = useState(false);
   const [goToDashboard, setGoToDashboard] = useState(false);
   const [goToEditEvent, setGoToEditEvent] = useState(false);
   const [eventToEdit, setEventToEdit] = useState("");
-  const [eventsList, setEventsList] = [];
```

Comment:
lets create a state for event object:   `const [event, setEvent] = useState("");`